playwright/_impl/_js_handle.py

```diff
@@ -205,7 +210,10 @@ def parse_value(value: Any, refs: Optional[Dict[int, Any]] = None) -> Any:
             return a
 
         if "d" in value:
-            return datetime.fromisoformat(value["d"][:-1])
+            # Node.js Date objects are always in UTC.
+            return datetime.datetime.strptime(
```

Comment:
It's unfortunate that we send an iso string over the wire, parsing millis since epoch (with `datetime.fromtimestamp`) would be more straightforward. But that's a big change.